File: ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { WebsiteAnalysisApis2SDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await WebsiteAnalysisApis2SDK.test()
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- WebsiteAnalysisApis2 SDK exists test

local sdk = require("website-analysis-apis2_sdk")

describe("WebsiteAnalysisApis2SDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
